files/ng-file-upload/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "ng-file-upload",
+  "repo": "danialfarid/ng-file-upload",
+  "files": {
+    "include": ["ng-file-upload-shim.min.js", "ng-file-upload-all.min.js", "ng-file-upload.min.js", "FileAPI.min.js"]
```

Comment:
Also I believe `FileAPI.flash.swf` is required as well in production. is it not?